src/coreclr/gc/env/gcenv.base.h

```diff
@@ -239,7 +243,7 @@ typedef DWORD (WINAPI *PTHREAD_START_ROUTINE)(void* lpThreadParameter);
 // but callers should only rely on it when the function returns TRUE;
 // otherwise, the stored value is undefined and varies by implementation
 // and hardware platform.
-inline uint8_t BitScanForward(uint32_t *bitIndex, uint32_t mask)
+inline uint8_t BitScanForward(DWORD *bitIndex, uint32_t mask)
```

Comment:
We have been trying to move away from Windows-specific types like `DWORD`. What is forcing us to go back to `DWORD` here?

Reply:
Thanks! This was to address a build error from an earlier iteration and no longer needed.